docs/api/tasks-ref-topic-get-task-by-title.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+---
+
+# Get task by title
+
+Returns an array of [`task`](task) objects only specified by the `title` parameter, if it exists.
+
+
+
+## URL
+
+```shell
+
+{GET}{server_url}/tasks/{title}
```

Comment:
```suggestion
{GET} {server_url}/tasks?title={title}
```

Check to see if the syntax is correct for the query.

Reply:
I checked. The syntax is correct.